### PR TITLE
Translate Ruby 2.4.0-rc1 Released (ko)

### PR DESCRIPTION
**Pull request by yous**
@ruby/www-ruby-lang-org-i18n-ko Review please.

### CODE REVIEW

File: ko/news/_posts/2016-12-12-ruby-2-4-0-rc1-released.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+---
+layout: news_post
+title: "루비 2.4.0-rc1 릴리스"
+author: "naruse"
+translator: "yous"
+date: 2016-12-12 00:00:00 +0900
```
Comment (yous) [inline ko/news/_posts/2016-12-12-ruby-2-4-0-rc1-released.md]: This line or the file name should be updated: https://github.com/ruby/www.ruby-lang.org/pull/1525/files/c7bf27de906286557221b27243c0c6d7d45cb166#diff-5691c787a3d551f315888cca9cb0ad15

Comment (stomar) [inline ko/news/_posts/2016-12-12-ruby-2-4-0-rc1-released.md]: I assume release date 2016-12-12 was intended and changed accordingly.

Comment (stomar) [inline ko/news/_posts/2016-12-12-ruby-2-4-0-rc1-released.md]: @yous naruse confirmed; you can change to 2016-12-12 **18**:00:00 +0900.

**Comment by marocchino** [comment]
Thanks :3

**Comment by stomar** [comment]
CI build failed, any idea why?

**Comment by riseshia** [comment]
I don't... is there any log?

**Comment by yous** [comment]
Maybe this was merged before CI run.

**Comment by stomar** [comment]
@marocchino: your above commit failed in CI, though it was a trivial change; any idea why? (There was no problem with the final commit that's in master.)

From the Travis-CI log:

```
$ git fetch origin +refs/pull/1526/merge:
fatal: Couldn't find remote ref refs/pull/1526/merge
Unexpected end of command stream
The command "eval git fetch origin +refs/pull/1526/merge:" failed. Retrying, 2 of 3.
```

**Comment by marocchino** [comment]
oops. I made a commit and merge without waiting for ci. sorry.

**Comment by stomar** [comment]
It was a trivial change, so I wouldn't mind in this case, just wondering why it failed.

And for squash commits, please avoid useless duplication of commit messages, like

```
Translate Ruby 2.4.0-rc1 Released (ko) (#1526)
    
    * Translate Ruby 2.4.0-rc1 Released (ko)
```

**Pull request merged**